include/dd/Package.hpp

```diff
@@ -1013,6 +1063,11 @@ namespace dd {
             }
 
             auto e = makeDDNode(w, edge, true);
+
+            //           if (r.p != nullptr && e.p != r.p){
+            //               std::cout << "Caching error detected in add" << std::endl;
+            //           }
+
```

Comment:
superfluous comments

Reply:
This is statement it actually quite confinient to debug the operation cache, which is why I would like to keep it. I added an explaining comment to it:
>//           if (r.p != nullptr && e.p != r.p){ // activate for debugging caching only
            //               std::cout << "Caching error detected in add" << std::endl;
            //           }

Reply:
Makes sense 👍🏼